site/docs/operations/docker.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Hazelcast Jet with Docker
+description: How to use Jet in a Docker environment
+---
+
+# Hazelcast Jet with Docker
+
+This section provides comprehensive coverage of usage of Hazelcast Jet
+inside a Docker environment. While it gives an explanation of all
+options and parameters used in the examples it is recommended to be
+familiar with Docker, visit e.g. the [official
+documentation](https://docs.docker.com/get-started/) for a beginner
+focused tutorial.
+
+# Start Hazelcast Jet node
+
+Let's begin with starting an instance of Hazelcast Jet using the
+official Docker image. To start the instance simply execute the
+following command in your terminal:
+
+```bash
+docker run hazelcast/hazelcast-jet
+```
+
+If you previously haven't run Hazelcast Jet Docker image, it will
+download the latest version of the image from [Docker
+hub](https://hub.docker.com/r/hazelcast/hazelcast-jet/). You can check
+the version of Jet in the logs, look for a line similar to this one:
+
+```text
+2020-05-18 19:34:43,872 [ INFO] [main] [c.h.system]: Hazelcast Jet 4.1 (20200429 - e6c60a1) starting at [172.17.0.2]:5701
+```
+
+You can also verify the version of Hazelcast Jet by running a following
+command without starting a full instance:
+
+```bash
+docker run hazelcast/hazelcast-jet jet --version
+```
+
+If your `hazelcast/hazelcast-jet` latest image points to an older
+version you can update it by running the following command:
+
+```bash
+docker pull hazelcast/hazelcast-jet
+```
+
+You should get output similar to:
+
+```text
+$ docker run hazelcast/hazelcast-jet jet --version
+Hazelcast Jet 4.1
```

Comment:
should we put the version tag here so it's auto-replaced? (+ all other places in the document)